website_partner_rating/README.md

```diff
@@ -0,0 +1,11 @@
+# website_partner_rating
```

Comment:
Hi @Amit-Parik 
Thanks for the contribution. 

Could you use a standard README file, following https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst